sdcclient/_scanning.py

```diff
@@ -291,6 +291,29 @@ def check_image_evaluation(self, image, show_history=False, detail=False, tag=No
 
         return [True, res.json()]
 
+    def get_pdf_report(self, image, pdfpath, tag=None, date=None):
```

Comment:
`get_pdf_report` is a little misleading. It'd be perfect (and very consistent with the rest of the library API) if the function wasn't also saving the PDF to disk.

I'm not sure if there's anything else you could do with the response of `/api/scanning/v1/images/{image_digest}/report`:
- if so, would it make more sense to split the `get` from the `save` to have 2 different functions?
- otherwise, then I'd just suggest to rename the most to something like `save_report_to_file` or similar

Reply:
Makes sense. I'll return the pdf bytes as the result and let the get_pdf_report users to handle the saving of the pdf into disk.